llvm/include/llvm/CodeGen/SelectionDAG.h

```diff
@@ -1130,6 +1130,9 @@ class SelectionDAG {
     return getNode(ISD::UNDEF, SDLoc(), VT);
   }
 
+  /// Return an POISON node. POISON does not have a useful SDLoc.
+  SDValue getPoison(EVT VT) { return getNode(ISD::POISON, SDLoc(), VT); }
```

Comment:
nit: Maybe since we're getting the actual ISD::POISIN, we should call it `getPOISON()` similar to the other nodes that get the actual ISD node?